lib/src/collection/treeset.dart

```diff
@@ -40,22 +40,22 @@ abstract class TreeSet<V> extends IterableBase<V> implements Set<V> {
   @override
   bool get isNotEmpty => length != 0;
 
-  /// Returns an [BidirectionalIterator] that iterates over this tree.
+  /// Returns an [TreeIterator] that iterates over this tree.
```

Comment:
nit (here and below): "a" TreeIterator now.

Reply:
Done.